components/Video/Video.js

```diff
@@ -70,6 +70,8 @@ class Video extends PureComponent {
     return (
       <View style={style.container}>
         <WebView
+          allowsFullscreenVideo
+          mediaPlaybackRequiresUserAction={false}
```

Comment:
Are we sure we want to have `mediaPlaybackRequiresUserAction` set to `false`?

I know everyone hates websites that autoplay videos, I doubt they have a different feeling towards mobile apps.

This would also be a change from our current usage, which isn't outlined anywhere in the issue description.

In my opinion, this shouldn't be changed or set to `false` in general.

Reply:
This is **not** autoplay option, even I understand why you'd think that, I've been there.
When user presses play button on Vimeo player, player goes full-screen. After video loads, player stops and user needs to press play again. This prop "fixed" that. Now, after user presses play in inline player, it goes full-screen, loads and starts playing.

Check "After video" for reference.

Reply:
Definitely agree with Vlad that we do not want to autoplay a video as soon as it renders on the screen, but Tomislav did show me the functionality and it doesn't do so. We do need to check whether this "broke" iOS behaviour, though. 

I don't understand why did we need to set it explicitly to false if its value is false by default?

Anyhow, the behavior we're seeking is:
1. when the user presses "play", the video is opened in fullscreen and starts playing
2. user can, at any time, switch to landscape mode (rotate their mobile phone) and back
3. user can, at any time, exit the fullscreen mode and then enter back in

Reply:
I'll run it on iOS and check if that's the behavior.

Reply:
I'm sorry. Its value is true by default.

iOS worky fine too, same as before. I should've attached iOS video too.
I'll do it soon, so you can see

Reply:
I'll take your word for it and accept responsibility if it's wrong.

Merging this and will begin release soon.

Reply:
😱